src/components/MainPane.svelte

```diff
@@ -289,7 +289,7 @@
   {#if settingsOpen}
     <slot name="settings" />
   {:else}
-    <div>
+    <div style="max-width: 100%; position: absolute;">
```

Comment:
Absolute just makes it look like this, even on bottom text direction:

![image](https://user-images.githubusercontent.com/20059164/144842980-6a3bbda9-6e5e-4467-846b-c77262e05566.png)

Reply:
This seems to be the test that failed. The tests actually caught something. I'm so proud.